containers/cpp-mariadb/.devcontainer/install-mariadb.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```

Comment:
Did we write this or do we need attribution and license reference here?

Reply:
We wrote this script as a whole.